### DISCUSSION TITLE
TODO LIST

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#187] [1. 개발 관련 글 모음]
https://velog.io/@shkisme/%EC%B1%85-%EB%B0%8F-%EA%B0%95%EC%9D%98-%EB%AA%A8%EC%9D%8C

요즘 글을 적기 힘드네요...
전체적으로 생산성이 떨어지고 있는 요즘입니다. 그런 의미에서 제가 공부하고 싶은 자료들을 정리라도 해 보았어요. 이 글은 계속 업데이트 할 예정입니다!

**Comment by pnudpwls1103** [comment]
공부할 양도 많고 분야도 되게 다양하네요..! 역시 제품을 만드는 사람은 알아야 할게 한두가지가 아닌거 같습니다 ㅎㅎ.. 화이띵!!!
